pkgs/top-level/all-packages.nix

```diff
@@ -34952,6 +34952,8 @@ with pkgs;
 
   lilypond = callPackage ../misc/lilypond { guile = guile_1_8; };
 
+  lilypond-unstable = callPackage ../misc/lilypond/unstable.nix { };
+
```

Comment:
This would be convenient.

```suggestion

lilypond-unstable-with-fonts = callPackage ../misc/lilypond/with-fonts.nix { lilypond = lilypond-unstable; };

```

Reply:
hmm.. but the fonts show up in the 2.22.2 dir

Reply:
updated the suggest with a dirty solution...

Reply:
Can't seem to directly commit this so I manually squashed this change into one commit. Nice work regardless!